### PR TITLE
Replace RTC image with illustration.

### PR DESCRIPTION
**Pull request by sausagejohnson**

### CODE REVIEW

File: settingup.tex [hunk 1]
```diff
@@ -102,7 +102,7 @@ \subsection{Replacing the Real-Time Clock Battery}
 
 The battery is located between the controller ports and the keyboard connector:
 
-\includegraphics[width=10cm]{images/rtc-battery-location}
+\includegraphics[width=\linewidth]{images/illustrations/rtc-battery-location.pdf}
```
Comment (gurcei) [inline settingup.tex:105]: The diagram did take up a big chunk of the page, and left that 'OPERATION' title at the bottom dangling a bit lonely:

image

Could we reduce the size a little, to let some of the following page's content squeeze into here too? Or alternatively, maybe we can force a new page to assure the 'OPERATION' title moves to the next page... Let's ponder it tomorrow.

But apart from that, nice 🙂

**Review by gurcei — CHANGES_REQUESTED** [review]

File: settingup.tex (hunk 1 above)
Comment (sausagejohnson) [reply settingup.tex:105]: Probably because I used the linewidth instead of 10cm. I'll reduce it back and see how it looks.

Comment (sausagejohnson) [reply settingup.tex]: All good to try again, @gurcei flow is restored using a 10cm width.

**Review by gurcei — APPROVED** [review]
Looking good, thanks man! :)

**Pull request merged**